docs/guides/tools.md

```diff
@@ -46,6 +46,7 @@ Tools are installed into temporary, isolated environments when using `uvx`.
     If you are running a tool in a [_project_](../concepts/projects.md) and the tool requires that your project is installed,
     e.g., when using `pytest` or `mypy`, you'll want to use [`uv run`](./projects.md#running-commands) instead of `uvx`.
```

Comment:
"e.g., when using `pytest` or `mypy` with your code in a `src/` directory"

Reply:
I think we should use the term 'flat project' instead of saying the `src` directory. If someone didn't know what it means like me they can search for it.

Reply:
Okay, that's fine with me — want to make that change here and we can merge your pull request?

Reply:
fine.

Reply:
done